channels/ke.m3u

```diff
@@ -1,17 +1,21 @@
 #EXTM3U
+#EXTINF:-1 tvg-id="Ebru TV Kenya" tvg-name="Ebru TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/download-1.jpg" tvg-url="" group-title="",Ebru TV Kenya
+https://tvsn-i.akamaihd.net/hls/live/261837/tvsn/tvsn_750.m3u8
 #EXTINF:-1 tvg-id="Ebru TV Kenya" tvg-name="Ebru TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2020/11/download-1.jpg" group-title="",Ebru TV Kenya
 https://stream-01.dc3.dailymotion.com/sec(yMMZhXcH3F9vUXdVEAQnDLr2SM0UWwYkuLICgaGcRKY)/dm/3/x67n3k1/s/live-2.m3u8
-#EXTINF:-1 tvg-id="Inooro TV Kenya" tvg-name="Inooro TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/InooroTV-1.jpg" group-title="",Inooro TV Kenya
+#EXTINF:-1 tvg-id="Inooro TV Kenya" tvg-name="Inooro TV" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/InooroTV-1.jpg" group-title="",Inooro TV Kenya [Expiring Token]
 https://stream-01.ix7.dailymotion.com/sec(IbeXB-fEZXft8Mu82uFOZQ89_KtucjBJ-iNVupUhcks)/dm/3/x7ttbvq/s/live-4.m3u8
+#EXTINF:-1 tvg-id="K24 Kenya" tvg-name="K24" tvg-country="KE" tvg-language="English" tvg-logo="https://kenyatv.tech/wp-content/uploads/2021/04/K24-1.jpg" tvg-url="" group-title="",K24 Kenya
+https://5f4db0f94b000.streamlock.net/k24/myStream/chunklist_w333466763.m3u8
```

Comment:
![image](https://user-images.githubusercontent.com/7253922/119133082-27034400-ba44-11eb-9728-33cf4369bbee.png)

Reply:
![image](https://user-images.githubusercontent.com/5430116/119220401-1067f600-baf3-11eb-905c-f302b32469b7.png)
It's back. Not 24/7?